### PR TITLE
Tweaks some codersprites

### PR DESCRIPTION
**Pull request by alsoandanswer**
dreammaker_YglC2ky5Rm
dreammaker_cHN0n1oTW9
dreammaker_3Si8s49HZQ
dreammaker_AtlejSOjE2

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
You might want to make every surgery tool a subtype of /surgery/ to remove all that copypaste and unnecessary for loops.

**Comment by alsoandanswer** [comment]
> You might want to make every surgery tool a subtype of /surgery/ to remove all that copypaste and unnecessary for loops.

That's a whole load of remapping for every single surgery tool and that's a can of worms I don't want to open.

**Comment by alsoandanswer** [comment]
!review

**Comment by KingOfThePing** [comment]
Will the ass still be visible with the new gowns? Can't see it quite right.

**Comment by alsoandanswer** [comment]
> Will the ass still be visible with the new gowns? Can't see it quite right.

It doesn't affect the worn site, just the ones when you put on the table

File: code/game/objects/items/weapons/surgery_tools.dm [hunk 1]
```diff
@@ -210,3 +221,30 @@
 		/obj/item/weapon/FixOVein = 1,
 		/obj/item/stack/medical/advanced/bruise_pack = 1,
 	)
+
+/obj/item/weapon/storage/fancy/tray/update_icon()
+	cut_overlays()
+
+	var/list/types_and_overlays = list(
+	/obj/item/weapon/bonesetter = "tray_bonesetter",
```
Comment (MarinaGryphon) [inline code/game/objects/items/weapons/surgery_tools.dm]: would you mind indenting these and putting the closing parenthesis on a new line?

Comment (alsoandanswer) [reply code/game/objects/items/weapons/surgery_tools.dm]: ok. it is done

File: code/game/objects/items/weapons/surgery_tools.dm [hunk 2]
```diff
@@ -46,6 +48,7 @@
 	desc = "This stops bleeding."
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "cautery"
+	var/overlay_state = "tray_cautery"
```
Comment (skull132) [inline code/game/objects/items/weapons/surgery_tools.dm]: Remove all of these variable definitions. They're unneeded now.

Comment (alsoandanswer) [reply code/game/objects/items/weapons/surgery_tools.dm]: Ok

**Comment by alsoandanswer** [comment]
suck my ass travis

**Pull request merged**